tests/queries/0_stateless/02381_parse_date_time_64_best_effort_us.sql

```diff
@@ -0,0 +1,22 @@
+SELECT 'parseDateTime64BestEffortUS';
```

Comment:
(Cosmetic: It would be nice if you could name the test file (and the reference file) after the function-under-test, i.e. `tests/queries/0_stateless/02381_parseDateTime64BestEffortUS.sql` This would then also be consistent with the existing test `01281_parseDateTime64BestEffort.sql`.)

Reply:
Done